### ISSUE TITLE
A11y_Visual Studio Code Client_Sep2021_Debug Adapter Protocol_Home Page_UseOfColor: Microsoft link looks like text , no way to differentiate it when hovered on it with mouse pointer there is no under line of change color or font size 

### ISSUE THREAD

**Issue by harshgangrade111** [microsoft/debug-adapter-protocol#213]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTag**:#A11yTCS;#DesktopWeb;#Win10;#WCAG1.4.1;#FTP;#STP;#A11yMAS;#ChromiumEdge;#A11ySev2;#P2_Visual Studio Code Clien_Sep2021;#Visual Studio Code Client;#UseOfColor;#P2_WCP;

## **Environment Details:**

Application Name: Visual Studio Code Client
#URL: https://microsoft.github.io/debug-adapter-protocol
Browser Versions: Microsoft Edge Version 93.0.961.38 (Official build) (64-bit)
Windows Version: Windows10

**Example Page URL**: https://microsoft.github.io/debug-adapter-protocol/

## **Repro Steps:**

1. Open the URL: https://microsoft.github.io/debug-adapter-protocol/
2. Debug Adapter Protocol home screen will get open.
3. Tab to footer section and hover over the Microsoft link and check.

## **Actual:**

Microsoft link looks like text , no way to differentiate it when hovered on it with mouse pointer there is no under line of change color or font size . Use of color should be different.

**Same issue is also observed with throughout the application.**

## **Expected:**

There should be under line change in color or font on hovering. Use of color should be different for all the links so than user can differentiate links.

## **User Impact:**

Users will not be able to know that it is a link or text.

## **Attachment:**
![#213](https://user-images.githubusercontent.com/60422426/133765643-a545602d-cced-46f1-83d6-39d322624460.gif)

**Comment by sana-ajani** [comment]
There is a hover when you put mouse over the Microsoft logo, the url is shown off to the side. The other three buttons (Star, Watch) also behave the same. 
![image](https://user-images.githubusercontent.com/9721254/135343549-9d30a58f-c76f-4b2a-affe-d6bd332ac280.png)

**Comment by NiharikaTipirishetty** [comment]
#P2_debug-adapter-protocol_Web_Sep2021

**Comment by prabhudevu** [comment]
I have verified the issue in the Env: https://microsoft.github.io/debug-adapter-protocol/,
Issue is fixed we are getting the URL on the bottom left side on the page when we hover mouse over "Microsoft" link. Added Attachment for the reference.

![microsoftlink](https://user-images.githubusercontent.com/98821792/178645824-097f8378-0994-403c-a5e1-444cc95a4ef5.gif)


![Screenshot (338)](https://user-images.githubusercontent.com/98821792/178646158-3ecea6ba-38ba-4d1f-9329-a230346656b0.png)

**Comment by prabhudevu** [comment]
#closed;

**Comment by 12-shweta** [comment]
Github Tags: #P2_debugadapterprotocol_Web_Sep2021;
